tests/test_bot_unlike.py

```diff
@@ -0,0 +1,155 @@
+    def test_unlike_medias(self, patched_time_sleep, media_ids, total, max_per_day):
+        self.bot.total['unlikes'] = total
+        self.bot.max_per_day['unlikes'] = max_per_day
+        for media_id in media_ids:
+            responses.add(
+                responses.POST, "{api_url}media/{media_id}/unlike/".format(
+                    api_url=API_URL, media_id=media_id
+                ), json="{'status': 'ok'}", status=200
+            )
+        broken_items = self.bot.unlike_medias(media_ids)
+        test_unliked = self.bot.total['unlikes'] == max_per_day
+        test_broken = len(broken_items) == len(media_ids) - (max_per_day - total)
+        assert (test_unliked and test_broken)
+
+    @responses.activate
+    @patch('time.sleep', return_value=None)
+    def test_unlike_media_comments(self, patched_time_sleep):
+        my_test_comment_items = []
+        results = 5
+        for i, _ in enumerate(range(results)):
+            my_test_comment_items.append(TEST_COMMENT_ITEM.copy())
+            my_test_comment_items[i]['pk'] = TEST_COMMENT_ITEM['pk'] + i
+            if i % 2:
+                my_test_comment_items[i]['has_liked_comment'] = False
+            else:
+                my_test_comment_items[i]['has_liked_comment'] = True
+        media_id = 1234567890
+        response_data = {
+            "caption": TEST_CAPTION_ITEM,
+            "caption_is_edited": False,
+            "comment_count": results,
+            "comment_likes_enabled": True,
+            "comments": my_test_comment_items,
+            "has_more_comments": False,
+            "has_more_headload_comments": False,
+            "media_header_display": "none",
+            "preview_comments": [],
+            "status": "ok"
+        }
+        responses.add(
+            responses.GET, '{api_url}media/{media_id}/comments/?'.format(
+                api_url=API_URL, media_id=media_id), json=response_data, status=200)
+        for my_test_comment_item in my_test_comment_items:
+            responses.add(
+                responses.POST, "{api_url}media/{comment_id}/comment_unlike/".format(
+                    api_url=API_URL, comment_id=my_test_comment_item['pk']
+                ), json="{'status': 'ok'}", status=200
+            )
+        broken_items = self.bot.unlike_media_comments(media_id)
+        assert broken_items == []
+
+    @responses.activate
+    @patch('time.sleep', return_value=None)
+    def test_unlike_user(self, patched_time_sleep):
+        unliked_at_start = self.bot.total['unlikes']
+        user_id = 1234567890
+        response_data = {
+            'status': 'ok',
+            'user': TEST_USERNAME_INFO_ITEM
+        }
+        responses.add(
+            responses.GET, '{api_url}users/{username}/usernameinfo/'.format(
+                api_url=API_URL, username=user_id
+            ), status=200, json=response_data)
+        my_test_photo_items = []
+        results = 5
+        for i, _ in enumerate(range(results)):
```

Comment:
I suggest this
```suggestion
        for i in range(results):
```
because you don't use elements